bin/welcome.js

```diff
@@ -0,0 +1,10 @@
+#! /usr/env/bin node
+
+const chalk = require( 'chalk' );
+
+console.log( chalk.cyan( '             _                           ' ) );
+console.log( chalk.cyan( '    ___ __ _| |_   _ _ __  ___  ___      ' ) );
+console.log( chalk.cyan( '   / __/ _\` | | | | | \'_ \\/ __|/ _ \\ ' ) );
+console.log( chalk.cyan( '  | (_| (_| | | |_| | |_) \\__ \\ (_) |  ' ) );
+console.log( chalk.cyan( '   \\___\\__,_|_|\\__, | .__/|___/\\___/ ' ) );
+console.log( chalk.cyan( '               |___/|_|                \n' ) );
```

Comment:
> why not just use a single multiline template literal?

Because I forgot that multiline template literals existed :)

> we have access to Unicode, we need more pizazz

Suit yourself! I don't know how did you folks generate the ASCII art in the first place, but you should be able to tweak it :)